src/Nethermind/Nethermind.JsonRpc/IJsonRpcConfig.cs

```diff
@@ -108,5 +108,10 @@ public interface IJsonRpcConfig : IConfig
                           "If this limit is exceeded on Http calls 503 Service Unavailable will be returned along with Json RPC error. " +
                           "Defaults to number of logical processes.")]
         int? EthModuleConcurrentInstances { get; set; }
+
+        [ConfigItem(
+            Description = "Max logged request size",
+            DefaultValue = "2000")]
+        int maxLoggedRequestSize { get; set; }
```

Comment:
maxLoggedRequestSize -> MaxLoggedRequestSize
let's make it nullable, with default null value